types/index.d.ts

```diff
@@ -4,6 +4,7 @@ export interface IRedirectlessConfig {
   client_id?: string,
   response_type?: string,
   onAuthorizationSuccess: (response: Response) => void,
+  onAuthorizationFailed: (response: Response) => void,
```

Comment:
shouldn't this be optional?

Reply:
whooops -- Fixed in https://github.com/pingidentity/pf-authn-js-widget/pull/100/commits/914a7d8b8d95530bbff0b9f859b70efbd9da37b2